src/index.ts

```
@@ -22,18 +27,31 @@ import { registerUsersTools } from "./tools/users.js";
+    } catch {
+      return undefined;
+    }
+  }
+  // Migrate from legacy plaintext file
   try {
-    const data = await fs.readFile(AUTH_INFO_PATH, "utf8");
-    return JSON.parse(data) as Record<string, unknown>;
+    data = await fs.readFile(LEGACY_AUTH_INFO_PATH, "utf8");
+    const parsed = JSON.parse(data) as Record<string, unknown>;
+    await writeAuthInfoSecure(JSON.stringify(parsed, null, 2));
+    await fs.unlink(LEGACY_AUTH_INFO_PATH).catch(() => {});
```

Biome (2.4.4)

[error] Unexpected empty block.

(lint/suspicious/noEmptyBlockStatements)